common/protob/messages-cardano.proto

```diff
@@ -129,6 +156,8 @@ message CardanoSignTx {
     // optional bytes metadata = 11;                            // replaced in Mary era with auxiliary data below
     optional uint64 validity_interval_start = 12;               // transaction validity start - added in allegra
     optional CardanoTxAuxiliaryDataType auxiliary_data = 13;    // transaction auxiliary data
+    // TODO GK maybe there should be a wrapper CardanoTxMintType so that it could be easily extended later if needed
+    repeated CardanoAssetGroupType mint = 14;                   // transaction mint tokens
```

Comment:
How are we going to pass on witness list? (I'm not sure if there will be just a single witness or a possibly longer list.)